.travis.yml

```diff
@@ -117,7 +117,14 @@ before_install:
       sudo apt-get install sagemath-upstream-binary;
     fi
 install:
-  - if [[ "${TEST_SAGE}" != "true" ]]; then
+#The install cycle below is to test installation on systems without setuptools.
+  - if [[ "${TEST_AUTOWRAP}" == "true" ]]; then
+      pip uninstall -y setuptools;
+      python setup.py install;
+      pip uninstall -y sympy;
+      pip install setuptools;
+      python setup.py install;
+  - else if [[ "${TEST_SAGE}" != "true" ]]; then
```

Comment:
should be elif. see: http://www.thegeekstuff.com/2010/06/bash-if-statement-examples/